src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -50,9 +51,14 @@ export default App.extend({
   onChangeSelected() {
     this.toggleBulkSelect();
   },
+  onChangeSearchQuery(state) {
+    this.currentSearchQuery = state.get('searchQuery');
+  },
   initListState() {
     const storedState = this.getState().getStore(this.worklistId);
 
+    this.setState({ searchQuery: this.currentSearchQuery });
```

Comment:
I originally had this at the top of the `initListState()` function like this:

```js
initListState() {
  this.setState({ searchQuery: this.currentSearchQuery });

  const storedState = this.getState().getStore(this.worklistId);

  . . .

}
```

But if we set the search query state before `const storedState = this.getState().getStore(this.worklistId);`, ` this.getState().getStore()` will return the state defaults instead of what was stored local storage.

Since `this.setState()` sets local storage:

```js
onChange() {
  store.set(this.getStoreKey(), omit(this.attributes, 'searchQuery'));
},
```